neore/scripts/package.sh

```diff
@@ -67,6 +67,10 @@ function download_flume(){
     rm -f ${FLUME_WO_TAR}/lib/libthrift-*.jar
     mv ${LIBTHRIFT_JAR} ${FLUME_WO_TAR}/lib
 
+    _log "### Disable httpclient and httpcore libraries distributed within apache-flume bundle... ###"
+    mv ${FLUME_WO_TAR}/lib/httpclient-4.2.1.jar ${FLUME_WO_TAR}/lib/httpclient-4.2.1.jar.old
```

Comment:
Yes, DynamoDB SDK needs http-client and http-core 4.3.1. Flume distribution comes with http-client and http-core 4.2.1, thus in order to avoid conflicts with the classpath (Java does not ensure the search for a library is always done in the same way) and to ensure getting the proper version of the libraries (as stated in the pom.xml), the version within the flume bundle must be removed or renamed.

The installation from sources has been properly modified as well.